README.md

```diff
@@ -142,7 +142,7 @@ npm install contentful-migration
 ### Usage as a library
 
 ```javascript
-const runMigration = require('contentful-migration/built/bin/cli').runMigration
+const runMigration = require('contentful-migration')
```

Comment:
I would avoid default export in favor of 

```
const { runMigration } = require('contentful-migration');
```

so we can make non-breaking changes if we want export something else

Reply:
:+1: